### PR TITLE
@ #11 | coverage increase 4% more

### PR DESCRIPTION
**Pull request by datphan**

### CODE REVIEW

File: flask_classful.py [hunk 1]
```diff
@@ -501,8 +501,8 @@ def get_true_argspec(method):
 
 def unpack(value):
     """Return a three tuple of data, code, and headers"""
-    if not isinstance(value, tuple):
-        return value, 200, {}
+    # if not isinstance(value, tuple):
```
Comment (hoatle) [inline flask_classful.py]: don't uncomment this as the function has support for not tuple value case.

Comment (datphan) [reply flask_classful.py]: they already checked it before, line `304`:
```
if isinstance(response, tuple):
                response, code, headers = unpack(response)
```

Comment (hoatle) [reply flask_classful.py]: they're different. this is what the function provides and that is how the function is used. The provided function cannot know how it is used in different use cases. For example, now I have another location where the provided function is used without tuple check, the provided function with your changes will be crashed.

**Comment by hoatle** [comment]
before:
```
dev_1  | Ran 173 tests in 2.622s
dev_1  | 
dev_1  | OK
dev_1  | coverage report --omit=test_classful/*,test_classful_py3/*,setup.py
dev_1  | Name                Stmts   Miss Branch BrPart  Cover
dev_1  | -----------------------------------------------------
dev_1  | flask_classful.py     270     14    141     11    94%
```
after:
```
dev_1  | Ran 178 tests in 2.002s
dev_1  | 
dev_1  | OK
dev_1  | coverage report --omit=test_classful/*,test_classful_py3/*,setup.py
dev_1  | Name                Stmts   Miss Branch BrPart  Cover
dev_1  | -----------------------------------------------------
dev_1  | flask_classful.py     270      9    141      7    96%
```

so 2% more test coverage is added.

**Pull request merged**